OpenTabletDriver.Web/Views/Wiki/FAQ/Windows.cshtml

```diff
@@ -113,35 +163,35 @@
 
 <hr/>
 
-<h3 class="wiki-nav-item pb-2" id="uac">
-    OpenTabletDriver gives an Administrator warning
+<h3 class="wiki-nav-item pb-2" id="administrator">
+    Plugin Manager won't open/I get a warning on startup?
 </h3>
 <div class="ms-3">
     <p>
-        When Starting OpenTabletDriver you may get an error warning you about Administrator Permissions. OpenTabletDriver 
-        shouldn't be run as Administrator. This causes the plugin manager, FAQ page and Tablet debugger to not work along with 
-        other features.
+        When Starting OpenTabletDriver you may get an error warning you about Administrator Permissions. OpenTabletDriver
+        shouldn't be run as Administrator. This causes the plugin manager, FAQ page and Tablet debugger to not work along with
+        other features.<br/><br>
+
+        The following instructions should fix your issue under normal circumstances, but if you're using the disabled by default account called "Administrator",
+        usually because you're using a "custom" iso (unofficial version of Windows) these instructions will not work for you.
```

Comment:
```suggestion
        usually because you're using a "custom" iso (unofficial version of Windows, also sometimes referred to as a "debloated windows") install, these instructions will not work for you. We generally recommend against using altered or 3rd party versions of windows, as OTD is likely to not be the only software adversely affected.
```